cocos/2d/renderer/batcher-2d.ts

```diff
@@ -971,7 +973,7 @@ class LocalDescriptorSet  {
     private _transformUpdate = true;
     private declare _localData: Float32Array | null;
 
-    public get descriptorSet (): DescriptorSet | null {
+    public getDescriptorSet (): DescriptorSet | null {
```

Comment:
Why have to change it?

Reply:
It's because we're using babel to compile ts to js.
babel has its own getter/setter accessor definition which could not be understood by `terser` minifier.

See this issue: https://github.com/terser/terser/issues/322

Since `LocalDescriptorSet` is an internal module, change the `get descriptSet` to function `getDescriptorSet` is fine for us.

Reply:
Ok, it is better to add a description.

Reply:
Added comments.